src/HIE/Bios/Environment.hs

```diff
@@ -309,5 +309,7 @@ anyToken :: ReadP Char
 anyToken = satisfy $ const True
 
 -- Used for clipping the trailing newlines on GHC output
+-- Also only take the last line of output
+-- (Stack's ghc output has a lot of preceding noise from 7zip etc)
 trim :: String -> String
-trim = dropWhileEnd isSpace
+trim = dropWhileEnd isSpace . last . lines
```

Comment:
This would crash the program if there is no output right?
I dont think it would be nice to have a crash there.